### ISSUE TITLE
Unable to Run ESLint Inside Docker Container

### ISSUE THREAD

**Issue by kcne** [openstreetmap/openstreetmap-website#4989]
### Problem

Error is encountered when attempting to run `bundle exec rails eslint` inside the Docker container, after following the instructions in the `DOCKER.md` file.

**Steps to Reproduce:**
1. Follow the setup instructions in the `DOCKER.md` file.
2. Open a terminal in the Docker container.
3. Run the command: `bundle exec rails eslint`.

**Observed Behavior:**
The following error is displayed:

```bash
$ bundle exec rails eslint
warning: parser/current is loading parser/ruby30, which recognizes 3.0.7-compliant syntax, but you are running 3.0.2.
Please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
yarn run v1.22.22
$ /app/node_modules/.bin/eslint -c /app/config/eslint.js /app/app/assets/config/manifest.js /app/app/assets/javascripts/application.js /app/app/assets/javascripts/auth_providers.js /app/app/assets/javascripts/diary_entry.js /app/app/assets/javascripts/fixthemap.js /app/app/assets/javascripts/id.js /app/app/assets/javascripts/index/changeset.js /app/app/assets/javascripts/index/contextmenu.js /app/app/assets/javascripts/index/directions/fossgis_osrm.js /app/app/assets/javascripts/index/directions/fossgis_valhalla.js /app/app/assets/javascripts/index/directions/graphhopper.js /app/app/assets/javascripts/index/directions.js /app/app/assets/javascripts/index/export.js /app/app/assets/javascripts/index/history.js /app/app/assets/javascripts/index/layers/data.js /app/app/assets/javascripts/index/layers/notes.js /app/app/assets/javascripts/index/new_note.js /app/app/assets/javascripts/index/note.js /app/app/assets/javascripts/index/query.js /app/app/assets/javascripts/index/search.js /app/app/assets/javascripts/index.js /app/app/assets/javascripts/leaflet.key.js /app/app/assets/javascripts/leaflet.layers.js /app/app/assets/javascripts/leaflet.locate.js /app/app/assets/javascripts/leaflet.map.js /app/app/assets/javascripts/leaflet.note.js /app/app/assets/javascripts/leaflet.query.js /app/app/assets/javascripts/leaflet.share.js /app/app/assets/javascripts/leaflet.sidebar-pane.js /app/app/assets/javascripts/leaflet.sidebar.js /app/app/assets/javascripts/leaflet.zoom.js /app/app/assets/javascripts/login.js /app/app/assets/javascripts/matomo.js /app/app/assets/javascripts/messages.js /app/app/assets/javascripts/oauth.js /app/app/assets/javascripts/richtext.js /app/app/assets/javascripts/router.js /app/app/assets/javascripts/social_share_button.js /app/app/assets/javascripts/user.js /app/app/assets/javascripts/welcome.js /app/config/eslint.js
internal/modules/cjs/loader.js:818
  throw err;
  ^

Error: Cannot find module 'node:util'
Require stack:
- /app/node_modules/eslint/bin/eslint.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
    at Function.Module._load (internal/modules/cjs/loader.js:667:27)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:85:18)
    at getErrorMessage (/app/node_modules/eslint/bin/eslint.js:67:18)
    at process.onFatalError (/app/node_modules/eslint/bin/eslint.js:123:3)
    at process.emit (events.js:314:20)
    at process._fatalException (internal/process/execution.js:165:25) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/app/node_modules/eslint/bin/eslint.js' ]
}
error Command failed with exit code 7.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
# 
```

**Expected Behavior:**
The `bundle exec rails eslint` command should run without error inside the Docker container.

**Environment:**
- Node.js version: v12.22.9
- Container OS: Ubuntu:22.04

**Cause:**
The issue is in the Node.js version not being specified. Installing Node.js alongside system packages as specified inside `Dockerfile` results in Docker installing Node.js v12, which is not compatible for running ESLint with our config.

**Comment by mmd-osm** [comment]
This issue has also been reported in https://github.com/openstreetmap/openstreetmap-website/issues/3275#issuecomment-1106929281

**Comment by mmd-osm** [comment]
Should be fixed now (#4992)
